packages/astro/src/assets/fonts/implementations/css-renderer.ts

```diff
@@ -0,0 +1,50 @@
+import type { CssProperties, CssRenderer } from '../definitions.js';
+
+export function renderFontFace(properties: CssProperties, minify: boolean): string {
+	// Line return
+	const lr = minify ? '' : `\n`;
+	// Space
+	const sp = minify ? '' : ' ';
+
+	return `@font-face${sp}{${lr}${Object.entries(properties)
+		.filter(([, value]) => Boolean(value))
+		.map(([key, value]) => `${sp}${sp}${key}:${sp}${value};`)
+		.join(lr)}${lr}}${lr}`;
+}
+
+export function renderCssVariable(key: string, values: Array<string>, minify: boolean): string {
+	// Line return
+	const lr = minify ? '' : `\n`;
```

Comment:
nit: `lf` it's usually called "line feed",  "new line", "line break" https://en.wikipedia.org/wiki/Newline